website/integrations/services/jellyfin/index.md

```diff
@@ -87,6 +89,107 @@ At this point, enter a username and click "Save Search Attribute Settings and Qu
 1. Click "Save"
 2. Logout, and login with a LDAP user. Username **must** be used, logging in with email will not work.
 
-## authentik Configuration
+### Authentik Configuration
 
 No additional authentik configuration needs to be configured. Follow the LDAP outpost instructions to create an LDAP outpost and configure access via the outpost
+
+## OIDC Configuration
+
+### Authentik Configuration
+
+To use the role claim within Jellyfin we will have to add a custom property. This is not needed but does allow admins to be set for SSO users automatically. It can also deny access if they don't have a role, but this should be done though Authentik instead.
+
+1. Go to _Customization_ then _Property Mappings_
+2. Create a _Scope Mapping_
+3. Assign these values:
+    - name: `Group Membership`
+    - scope name: `groups`
+    - description: `See which groups you belong to`
+    - expression: `return [group.name for group in user.ak_groups.all()]`
+
+![](./jellyfin_mapping.png)
+
+**Provider Settings**
+
+In authentik under _Providers_, create an _OAuth2/OpenID Provider_ with these settings:
+
+-   Name: `jellyfin`
+-   Redirect URI: `https://jellyfin.company.com/sso/OID/redirect/authentik`
+-   Signing Key: Select any available key
+
+For the Group Membership we just created go into _Advanced Protocol Settings_ and add that Group Membership to the selected.
```

Comment:
```suggestion
For the Group Membership we just created go into **Advanced Protocol Settings** and add that Group Membership to the selected.
```

Reply:
@Redlonghead to what exactly are they adding the Group Membership to...? "To the selcted..."?

Reply:
My bad they are adding the scope called "Group Membership" to the scope under **Advanced Protocol Settings**. I had it sown in the photo but didn't specify it in the actual documentation